testing/cpm/cpm_find-patch-command/CMakeLists.txt

```diff
@@ -0,0 +1,49 @@
+#=============================================================================
+# Copyright (c) 2023-2024, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#=============================================================================
+cmake_minimum_required(VERSION 3.23.1)
+project(rapids-cpm_find-patch-command-project LANGUAGES CXX)
+
+set(deps_dir "${CMAKE_CURRENT_BINARY_DIR}/_cccl_dep")
+if(NOT EXISTS "${deps_dir}")
+  file(MAKE_DIRECTORY "${deps_dir}")
+  find_package(Git)
+  execute_process(
+    COMMAND ${GIT_EXECUTABLE} clone --depth 1 --branch v2.2.0 https://github.com/NVIDIA/cccl.git
```

Comment:
I expect that this test will fail unless we keep the `v2.2.0` in sync with the `versions.json`, because any other version will fail to apply the patches when it calls `rapids_cpm_cccl()`. Is that correct? Is there anything we can do to avoid needing to modify this test's CCCL version in the future?

Reply:
Yes, thanks for the reminder. I meant to fix this earlier, it's updated now.